### PR TITLE
Replace ts() to E::ts()

### PR DESCRIPTION
**Pull request by lucky091588**

### CODE REVIEW

**Comment by lucky091588** [comment]
I already check this and there is not E error.

File: jsumfields.php [hunk 1]
```diff
@@ -2150,7 +2151,7 @@ function _jsumfields_generate_data_based_on_current_data($session = NULL) {
     $session = CRM_Core_Session::singleton();
   }
   if (empty($sumfieldsCustomTableName)) {
-    $session::setStatus(ts("Your configuration may be corrupted. Please disable and renable this extension."), ts('Error'), 'error');
+    $session::setStatus(ts("Your configuration may be corrupted. Please disable and renable this extension."), E::ts('Error'), 'error');
```
Comment (twomice) [inline jsumfields.php:2154]: @lucky091588 Looks like there's one more `ts()` that needs attention here. Please fix?

**Comment by lucky091588** [comment]
@twomice Done updating the file that I miss on the changes.

**Comment by twomice** [comment]
Merged, thanks @lucky091588

**Pull request merged**